packages/react-components/react-breadcrumb/src/components/BreadcrumbDivider/useBreadcrumbDividerStyles.styles.ts

```diff
@@ -9,9 +9,19 @@ export const breadcrumbDividerClassNames: SlotClassNames<BreadcrumbDividerSlots>
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
-  root: {
-    display: 'flex',
+const useStyles = makeResetStyles({
+  display: 'flex',
+});
+
+const useIconStyles = makeStyles({
+  small: {
+    fontSize: '12px',
```

Comment:
where are these sizes coming from ?

do we have a generalized framework based on design spec for these sizes ? or its case base case ?

Reply:
it's according to the design spec